ping.go

```diff
@@ -66,14 +65,15 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/google/uuid"
 	"golang.org/x/net/icmp"
 	"golang.org/x/net/ipv4"
 	"golang.org/x/net/ipv6"
 )
 
 const (
 	timeSliceLength  = 8
-	trackerLength    = 8
+	trackerLength    = 16
```

Comment:
maybe:

```suggestion
	trackerLength    = len(uuid.UUID{})
```

as the expression ` len(uuid.UUID{})` is a compile-time constant.

Reply:
Good idea.